app/init/launch.ts

```diff
@@ -219,3 +220,9 @@ export const getLaunchPropsFromNotification = async (notification: NotificationW
 
     return launchProps;
 };
+
+// Ignore all notifications if running e2e
+const isRunningE2e = RUNNING_E2E === 'true';
+// eslint-disable-next-line no-console
+console.log(`RUNNING_E2E: ${RUNNING_E2E}, isRunningE2e: ${isRunningE2e}`);
+LogBox.ignoreAllLogs(isRunningE2e);
```

Comment:
I think this should be moved to the index file where other logBox ignores are being set.. the good thing with this is that we won't need to handle warnings issued by adding an empty array to the operators.

Reply:
@enahum Moved the block to `index.ts` and fixed associated imports